.github/workflows/beam_PostCommit_Python_Nexmark_Direct.yml

```diff
@@ -0,0 +1,134 @@
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: PostCommit Python Nexmark Direct
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: '0 */6 * * *'
+  workflow_dispatch:
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.issue.number || github.event.pull_request.head.label || github.sha || github.head_ref || github.ref }}-${{ github.event.schedule || github.event.comment.body || github.event.sender.login}}'
+  cancel-in-progress: true
+
+#Setting explicit permissions for the action to avoid the default permissions which are `write-all` in case of pull_request_target event
+permissions:
+  actions: write
+  pull-requests: read
+  checks: read
+  contents: read
+  deployments: read
+  id-token: none
+  issues: read
+  discussions: read
+  packages: read
+  pages: read
+  repository-projects: read
+  security-events: read
+  statuses: read
+
+env:
+  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
+  GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GE_CACHE_USERNAME }}
+  GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GE_CACHE_PASSWORD }}
+  GRADLE_JAVA_COMMAND_ARGUMENTS: |
+    --manageResources=false
+    --monitorJobs=true
+    --bigQueryTable=nexmark
+    --bigQueryDataset=nexmark
+    --project=apache-beam-testing
+    --resourceNameMode=QUERY_RUNNER_AND_MODE
+    --exportSummaryToBigQuery=true
+    --tempLocation=gs://temp-storage-for-perf-tests/nexmark
+    --influxDatabase=beam_test_metrics
+    --influxHost=http://10.128.0.96:8086
+    --baseInfluxMeasurement=nexmark
+    --exportSummaryToInfluxDB=true
+    --influxRetentionPolicy=forever
+    --suite=SMOKE
+    --enforceEncodability=true
+    --enforceImmutability=true
+    --runner=DirectRunner
+    --numEvents=100000
+  GRADLE_PYTHON_COMMAND_ARGUMENTS: |
+    --monitor_jobs
+    --big_query_table=nexmark
+    --big_query_dataset=nexmark
+    --project=apache-beam-testing
+    --resource_name_mode=QUERY_RUNNER_AND_MODE
+    --export_summary_to_big_query
+    --temp_location=gs://temp-storage-for-perf-tests/nexmark
+    --influx_database=beam_test_metrics
+    --influx_host=http://10.128.0.96:8086
+    --base_influx_measurement=nexmark
+    --export_summary_to_influx_db
+    --influx_retention_policy=forever
+    --suite=SMOKE
+    --enforce_encodability
+    --enforce_immutability
+    --runner=DirectRunner
+    --num_events=100000
+
+jobs:
+  beam_PostCommit_Python_Nexmark_Direct:
+    name: ${{matrix.job_name}} (${{matrix.job_phrase}})
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 240
+    strategy:
+      fail-fast: false
+      matrix:
+        job_name: [beam_PostCommit_Java_Nexmark_Direct]
+        job_phrase: [Run Python Direct Runner Nexmark Tests]
+        query: [0, 2, 3, 5, 7, 8, 10, 11]
```

Comment:
Could we add the links we have here for context on why we're only running some of the queries - https://github.com/apache/beam/blob/1f980eaa894cc43ea5ca1aeb4cb2ef1de1162b17/.test-infra/jenkins/NexmarkBuilder.groovy#L187

Reply:
Done, thanks!

Reply:
Sorry, I meant the github issue links that are referenced in that file. I think the Jenkins file should go away eventually, so we should duplicate the github issue links here.

Reply:
Updated